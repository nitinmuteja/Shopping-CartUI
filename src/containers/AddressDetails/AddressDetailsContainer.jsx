import React from 'react';
import Container from '@material-ui/core/Container';
import TextField from '@material-ui/core/TextField';
import Select from '@material-ui/core/Select';
import MenuItem from '@material-ui/core/MenuItem';
import InputLabel from '@material-ui/core/InputLabel';
import FormControl from '@material-ui/core/FormControl';
import Box from '@material-ui/core/Box';
import Button from '@material-ui/core/Button';



class AddressDetailsContainer extends React.Component {


    render() {
        return (
        <Container fixed>
        <Box mt={4}>
        <Box component="div" display="inline" fontWeight="fontWeightBold" fontSize="h4.fontSize">Select delivery details to continue.</Box>
        </Box>
          <Box mt={4} >
            <TextField id="customer-name" label="Name" margin="normal" fullWidth />
          </Box>

          <TextField id="customer-address" label="Address" multiline rowsMax="4" fullWidth /> <br/>

          <Box mt={4}>
          <FormControl fullWidth>
            <InputLabel htmlFor="age-simple">Country</InputLabel>
            <Select  value="india" inputProps={{ name: 'age', id: 'age-simple', }} >
              <MenuItem value={"india"}>India</MenuItem>
              <MenuItem value={"nepal"}>Nepal</MenuItem>
              <MenuItem value={"bhutan"}>Bhutan</MenuItem>
            </Select>

            <Box mt={4}>
            <Button variant="contained" color="primary"  fullWidth> Continue </Button>
            </Box>

          </FormControl>
          </Box>

        </Container>
      )
    }

}
export default AddressDetailsContainer;
