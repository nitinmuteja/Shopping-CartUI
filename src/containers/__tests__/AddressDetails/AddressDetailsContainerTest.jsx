import React from 'react';
import { render } from '@testing-library/react';

import AddressDetailsContainer from "../../AddressDetails/AddressDetailsContainer";



describe('Address Details Component', () => {


    it('should match snapshot', () => {
        const { container } = render(<AddressDetailsContainer />);

        expect(container).toMatchSnapshot();
    });

});
