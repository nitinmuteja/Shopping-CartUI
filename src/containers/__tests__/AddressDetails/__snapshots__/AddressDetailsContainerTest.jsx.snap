// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address Details Component should match snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
  >
    <div
      class="MuiBox-root MuiBox-root-9"
    >
      <div
        class="MuiBox-root MuiBox-root-10"
      >
        Select delivery details to continue.
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-11"
    >
      <div
        class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink="false"
          for="customer-name"
        >
          Name
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="customer-name"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
    <div
      class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink="false"
        for="customer-address"
      >
        Address
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-multiline MuiInput-multiline"
      >
        <textarea
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input MuiInputBase-inputMultiline MuiInput-inputMultiline"
          id="customer-address"
          rows="1"
          style="height: 0px; overflow: hidden;"
        />
        <textarea
          aria-hidden="true"
          class="MuiInputBase-input MuiInput-input MuiInputBase-inputMultiline MuiInput-inputMultiline"
          readonly=""
          style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; width: 100%;"
          tabindex="-1"
        />
      </div>
    </div>
     
    <br />
    <div
      class="MuiBox-root MuiBox-root-66"
    >
      <div
        class="MuiFormControl-root MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink="true"
          for="age-simple"
        >
          Country
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <div
            aria-haspopup="listbox"
            class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
            id="select-age"
            role="button"
            tabindex="0"
          >
            India
          </div>
          <input
            id="age-simple"
            name="age"
            type="hidden"
            value="india"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSelect-icon"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
        </div>
        <div
          class="MuiBox-root MuiBox-root-88"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-fullWidth"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
               Continue 
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
